tarantool/README.md

```diff
@@ -0,0 +1,62 @@
+# Tarantool
+
+## Introduction
+
+Tarantool is a NoSQL In-Memory database.
+It's distributed under BSD licence and is hosted on [github][tarantool-github].
+
+Tarantool features:
+
+* Defferent index types with iterators:
+	- HASH (the fastest)
+	- TREE (range and ordered retreival)
+	- BITSET (bit mask search)
+	- RTREE (geo search)
+* multipart keys for HASH and TREE indexes
+* Data persistence with by Write Ahead Log (WAL) and snapshots.
+* asynchronous master-master replication, hot standby.
+* coroutines and async. IO are used to implement high-performance lock-free access to data.
+  - socket-io/file-io with yeilds from lua
+* stored procedures in Lua (Using LuaJIT)
+* supports plugins written on C/C++ (Have two basic plugins for working with MySQL and PostgreSQL)
+* Authentication and access control
+
+## Quick start
+
+This section descrives how to run YCSB against a local Tarantool instance
+
+### 1. Start Tarantool
+
+First, clone Tarantool from it's own git repo and build it (described in our [README.md][tarantool-readme]):
+
+    cp YCSB/tarantool/config/tarantool-tree.lua <vardir>/tarantool.lua
```

Comment:
this refers to the wrong location now. in the YCSB binary artifact, the location will be %YCSB%/tarantool/conf/tarantool-tree.lua